src/meshapi/tests/test_views_explorer.py

```diff
@@ -0,0 +1,28 @@
+from django.contrib.auth.models import Group, User
+from django.test import TestCase
+
+from meshapi.tests.group_helpers import create_groups
+
+
+class TestViewsGetExplorer(TestCase):
```

Comment:
Should we add a case for an unauthenticated user? 

Also maybe a simple check that something like `SELECT * FROM meshapi_members` gives HTTP 200? Just to make sure we catch any regressions of the permissions error you were getting at the start of this feature. I could imagine someone adding a new table and forgetting to exclude it in `settings.py` for example. Would be good to give them a red X on their PR

Reply:
Getting some weird-ass errors when I try to test this:
```

Traceback (most recent call last):
  File "/home/wilnil/Code/nycmeshnet/meshdb/.venv/lib/python3.12/site-packages/asgiref/local.py", line 89, in _lock_storage
    asyncio.get_running_loop()
RuntimeError: no running event loop

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/wilnil/Code/nycmeshnet/meshdb/.venv/lib/python3.12/site-packages/django/utils/connection.py", line 58, in __getitem__
    return getattr(self._connections, alias)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/wilnil/Code/nycmeshnet/meshdb/.venv/lib/python3.12/site-packages/asgiref/local.py", line 118, in __getattr__
    return getattr(storage, key)
           ^^^^^^^^^^^^^^^^^^^^^
AttributeError: '_thread._local' object has no attribute 'r'
```

For what it's worth, the library has some pretty extensive tests of its own: https://github.com/explorerhq/sql-explorer/blob/master/explorer/tests/test_views.py

Reply:
OK, because we're doing raw sql query shit, the tests for this end up positively awful because Django is so goddamn insistent on its concept of a `test_*` database. I have spent hours fighting with the tooling and all I have accomplished is breaking the everliving shit out of everything.

Reply:
OK, there, I got it: https://docs.djangoproject.com/en/5.1/topics/testing/advanced/#testing-primary-replica-configurations

don't make me do anything else.

Reply:
ope and even though it #worksonmymachine it doesn't work in github for some alien reason

Reply:
Yeah it’s crashing quite spectacularly